.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,171 @@
+    - name: Run Lighthouse CI
+      run: |
+        npm install -g @lhci/cli@0.13.x
+        lhci autorun
+      env:
+        LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
+
+    - name: Upload Lighthouse results
+      uses: actions/upload-artifact@v4
+      with:
+        name: lighthouse-results
+        path: .lighthouseci
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)